src/components/Single-Article.jsx

```diff
@@ -24,20 +26,26 @@ export const SingleArticle = () => {
         })
     }, [article_id])
     
-    console.log('1.', comments);
+    const handleVote = (article_id, vote) => {
+            patchVotes(article_id, vote);
+            setVoteChanges(prev => ({...prev, [article_id]: vote}));
```

Comment:
currently this is happening optimistically which is great! but we also want to make sure that if anything goes wrong and the vote isnt added to the db then we want to make sure to undo the optimistic render? where would be the best place to put this?